platforms/core-execution/execution/build.gradle.kts

```diff
@@ -4,27 +4,14 @@ plugins {
 
 description = "Execution engine that takes a unit of work and makes it happen"
 
-errorprone {
-    disabledChecks.addAll(
-        "AnnotateFormatMethod", // 1 occurrences
-        "BadImport", // 2 occurrences
-        "Finally", // 2 occurrences
-        "ReferenceEquality", // 1 occurrences
-        "SameNameButDifferent", // 5 occurrences
-        "StringCaseLocaleUsage", // 8 occurrences
-        "SuperCallToObjectMethod", // 2 occurrences
-        "UndefinedEquals", // 1 occurrences
-        "UnusedVariable", // 1 occurrences
-    )
-}
-
 dependencies {
     api(libs.guava)
     api(libs.jsr305)
     api(libs.slf4jApi)
 
     api(projects.concurrent)
     api(projects.javaLanguageExtensions)
+    api(libs.errorProneAnnotations)
```

Comment:
❓ Can this annotations be `compileOnly`?

Reply:
Good point, fixed